public/locales/en/translation.json

```diff
@@ -307,5 +307,26 @@
     "contactUsModal": {
       "message": "Thank you for leaving us message!"
     }
+  },
+  "signIn": {
+    "header": "Welcome Back!",
+    "SignInWith": "Sign in with",
+    "newHere": "New here? ",
+    "createAccount": "Create an account",
+    "form": {
+      "yourEmail":"Your email",
+      "yourPassword":"Your password",
+      "rememberMe":"Remember me next time",
+      "signIn":"Sign in"
+    },
+    "validationMessage": {
+      "incorrectPassword":"The password that you've entered is incorrect",
+      "emailNotFound":"The email address that you've entered doesn't match any account. Sign up for an account.",
+      "emptyEmail":"Please provide your email",
+      "emailNotValid":"Please provide a valid email",
+      "emptyPassword":"Please provide a password",
+      "passwordMin":"Password length must be at least 8 characters long",
+      "passwordMax":"Password length must be less than or equal to 64 characters long"
+    }
```

Comment:
the need space

Reply:
I thought that `yarn fix` would do that for me :(